centralserver/central/api_urls.py

```diff
@@ -29,9 +28,6 @@
 urlpatterns += patterns('centralserver.i18n.api_views',
     url(r'^i18n/', include(centralserver.i18n.api_urls)),
 )
-urlpatterns += patterns('centralserver.khanload.api_views',
-    url(r'^khanload/', include(centralserver.khanload.api_urls)),
-)
```

Comment:
It looks like we still have all the client-side stuff for KA data import in the distributed server:
https://github.com/learningequality/ka-lite/blob/0.16.x/kalite/contentload/api_views.py
but the API endpoints are no longer mounted anywhere.

And we even still have a UI element pointing to it:
https://github.com/learningequality/ka-lite/blob/0.16.x/kalite/control_panel/templates/control_panel/account_management.html#L42
But that whole `{% block titlebar_title %}` block is orphaned code, since in the parent template it's defined inside the content block, which is fully overridden below in `account_management.html`, so in practice the button is no longer ever seen.

So, we should be safe to remove it here, despite all the remaining cruft on the distributed server.